package.json

```diff
@@ -6,9 +6,7 @@
     "sourcemap",
     "vlq"
   ],
-  "main": "dist/sourcemap-codec.umd.js",
```

Comment:
this would technically be a breaking change for node versions that don't understand exports map. (They would fall back to /index.js i believe)